src/cpp/shared/game/game.h

```diff
@@ -162,7 +162,7 @@ class game {
 	char field_660D;
 	char field_660E;
 
-  // New state
+  // New states
```

Comment:
Please revert this

Reply:
Ah, this was left behind from when I was using a boolean checker grid for the WillJoin implementation. That array was declared here in a similar fashion, so it made sense to me at the time to pluralize the comment. I am undoing this now.